assets/src/design-system/components/tooltip/index.js

```diff
@@ -69,6 +69,8 @@ function Tooltip({
   onPointerLeave = () => {},
   onFocus = () => {},
   onBlur = () => {},
+  forceAnchorRef = null,
+  tooltipProps = null,
```

Comment:
@BrittanyIRL, I added these two props to the tooltip component, but they shouldn't have a big impact if not used. Both were needed for the slider component to first a) use a different reference point for the base position of the tooltip and b) move the tooltip dynamically as something else changes.

Reply:
thanks for the heads up! makes sense. happy to be a second reviewer. just tag me for review if you want more eyes.